internal/stage_match_alphanumeric.go

```diff
@@ -8,7 +8,17 @@ func testMatchAlphanumeric(stageHarness *test_case_harness.TestCaseHarness) erro
 	testCases := []TestCase{
 		{
 			Pattern:          `\w`,
-			Input:            "word",
+			Input:            "Word",
```

Comment:
@andy1li let's randomize more in this stage - since it only checks alphanumeric, it's pretty easy to randomize. We don't randomize in some other stages only because the regexes are complex and hard to generate with random inputs.